config/locales/uk.yml

```diff
@@ -0,0 +1,49 @@
+uk:
+  errors:
+    messages:
+      taken_in_past: 'раніше використовувався.'
+      equal_to_current_password: 'має відрізнятися від поточного паролю.'
+      password_complexity:
+        digit:
+          one: 'повинен включати хоча б одну цифру'
+          few: 'повинен включати хоча б %{count} цифри'
+          many: 'повинен включати хоча б %{count} цифр'
+          other: 'повинен включати хоча б %{count} цифри'
+        lower:
+          one: 'повинен включати хоча б одну малу букву'
+          few: 'повинен включати хоча б %{count} малі букви'
+          many: 'повинен включати хоча б %{count} малих букв'
+          other: 'повинен включати хоча б %{count} малі букви'
+        symbol:
+          one: 'повинен включати хоча б один розділовий знак або символ'
+          few: 'повинен включати хоча б %{count} розділові знаки або символи'
+          many: 'повинен включати хоча б %{count} розділових знаків або символів'
+          other: 'повинен включати хоча б %{count} розділові знаки або символи'
+        upper:
+          one: 'повинен включати хоча б одну прописну букву'
+          few: 'повинен включати хоча б %{count} прописні букви'
+          many: 'повинен включати хоча б %{count} прописних букв'
+          other: 'повинен включати хоча б %{count} прописні букви'
+  devise:
+    invalid_captcha: 'Неправильне введення капчі.'
+    invalid_security_question: 'Неправильна відповідь на секретне питання.'
+    paranoid_verify:
+      code_required: 'Пожалуйста введите код, полученный от нашей команды поддержки'
+      code_required: 'Введіть, будь ласка, код від нашої команди підтримки'
+    paranoid_verification_code:
+      show:
+        submit_verification_code: Відправити код підтвердження
```

Comment:
Again, pretty sure this is Google Translate not working well - it's translating this to "Send confirmation code." Would this be understood to be mean "submit?" 

When I think "send" I tend to think more like "send an email" versus "submit a form." 

Again, I'm pretty sure this is just an issue with Google translate, but I figured I'd ask.

Reply:
yes, it will be understandable correctly